src/main/groovy/lang/ObjectRange.java

```diff
@@ -338,22 +337,22 @@ public int size() {
 
         // Performance detail:
         // not using get(fromIndex), get(toIndex) in the following to avoid stepping over elements twice
-        final Iterator<Comparable> iter = new StepIterator(this, 1);
+        final Iterator<Comparable> it = new LazySteppingIterator(this, 1);
```

Comment:
why bloating the pull request by renaming iter to it? We tend to use iter for iterator, since "it" is often used as iteration element in Closures